assets/js/block-variation.js

```diff
@@ -0,0 +1,34 @@
+wp.blocks.registerBlockVariation( 'core/query', {
+	name: 'twentytwentyfour/featured-image-query',
+	title: 'Featured Images',
+	description: 'List of posts with featured images',
+	attributes: {
+		namespace: 'twentytwentyfour/featured-image-query',
+		query: {
+			perPage: 6,
+			postType: 'post',
+			order: 'desc',
+			orderBy: 'date',
+			inherit: false,
+		},
+		align: 'full',
+	},
+	allowedControls: [ 'order' ],
+	innerBlocks: [
+		[
+			'core/post-template',
+			{ layout: {
+				type: 'grid',
+				columnCount: 3
+			} },
+			[
+				[
+					'core/post-featured-image',
+					{
+						aspectRatio: 2/3,
```

Comment:
I'm not sure this is working correctly perhaps it should be `aspectRatio: '2/3'`,